crates/extensions/aws_mapper_ext/src/converter.rs

```diff
@@ -41,44 +46,77 @@ impl AwsConverter {
         if is_bridge_health(&input.topic.name) {
             Ok(vec![])
         } else {
-            let default_timestamp = self.add_timestamp.then(|| self.clock.now());
+            match self.mqtt_schema.entity_channel_of(&input.topic) {
+                Ok((source, channel)) => self.try_convert_te_topics(source, channel, input),
+                Err(_) => self.try_convert_tedge_topics(input),
+            }
+        }
+    }
+
+    fn try_convert_tedge_topics(
+        &mut self,
+        input: &MqttMessage,
+    ) -> Result<Vec<MqttMessage>, ConversionError> {
+        let default_timestamp = self.add_timestamp.then(|| self.clock.now());
+
+        // serialize with ThinEdgeJson for measurements, for alarms and events just add the timestamp
+        let payload = if input.topic.name.starts_with("tedge/measurements") {
```

Comment:
Why do you keep the check for `tedge/measurements`?

Reply:
I removed it, this is not necessary any more.